tags.go

```diff
@@ -7,45 +7,50 @@ type Tags string
 
 // NewTags combines a list of tags into a single shopify tag string
 func NewTags(tags []string) Tags {
-	return Tags(strings.Join(tags, ", "))
+	return Tags(strings.Join(tags, ","))
```

Comment:
Why are we removing the space here? I believe Shopify uses spaces after the comma 🙂

Reply:
It's a good point. I mean the description says.

```
A string of comma-separated tags that are used for filtering and search. A product can have up to 250 tags. Each tag can have up to 255 characters.
```

It does not specify you need spaces. If you look what it returns then it has spaces but if you look when creating a product it has no spaces as shown below.

```
'{"product":{"title":"Burton Custom Freestyle 151","body_html":"<strong>Good snowboard!</strong>","vendor":"Burton","product_type":"Snowboard","tags":["Barnes & Noble","Big Air","John's Fav"]}}'
```

I thought that if I just simplify it in the package it can stay consistent and when you split you don't have spaces so each element in the array has just the tag strip trimming the edges of white space.

Reply:
Ah yes you're right - I just tried hitting the endpoint without spaces and it looked fine! It's all good then 😄